Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -4568,4 +4572,97 @@ RefPtr<CSSStyleValue const> Parser::parse_filter_value_list_value(TokenStream<Co
     return FilterValueListStyleValue::create(move(filter_value_list));
 }
 
+RefPtr<CSSStyleValue const> Parser::parse_contain_value(TokenStream<ComponentValue>& tokens)
+{
+    // none | strict | content | [ [size | inline-size] || layout || style || paint ]
```

Comment:
I think you can simplify this by doing something like:
```c++
auto transaction = tokens.begin_transaction();
tokens.discard_whitespace();

// none
if (auto none = parse_all_as_single_keyword_value(tokens, Keyword::None)) {
    transaction.commit();
    return none;
}

// strict
if (auto strict = parse_all_as_single_keyword_value(tokens, Keyword::Strict)) {
    transaction.commit();
    return strict;
}

// ... etc
```

Reply:
So it seems. Didn't know about `parse_all_as_single_keyword_value()`, that's useful.